quickproject.lisp

```diff
@@ -60,34 +45,27 @@ not already exist."
 (defun current-year ()
   (nth-value 5 (decode-universal-time (get-universal-time))))
 
-(defun file-comment-header (stream)
-  (format stream ";;;; ~A~%" (file-namestring stream))
-  (when *include-copyright*
-    (format stream ";;;;~%")
-    (format stream ";;;; Copyright (c) ~D ~A~%" (current-year) *author*))
-  (terpri stream))
-
-(defun write-system-file (name file &key depends-on)
-  (with-new-file (stream file)
-    (file-comment-header stream)
-    (write-system-form name
-                       :depends-on depends-on
-                       :stream stream)
-    (terpri stream)))
-
-(defun write-application-file (name file)
-  (with-new-file (stream file)
-    (file-comment-header stream)
-    (format stream "(in-package ~S)~%~%" (uninterned-symbolize name))
-    (format stream ";;; ~S goes here. Hacks and glory await!~%~%" name)))
-
 (defvar *after-make-project-hooks* nil
   "A list of functions to call after MAKE-PROJECT is finished making a
 project. Each function is called with the same arguments passed to
 MAKE-PROJECT, except that NAME is canonicalized if
 necessary. *DEFAULT-PATHNAME-DEFAULTS* bound to the newly created
 project directory.")
 
+(defun template-pathname->output-name (path)
+  (flet ((mk-path (name)
+	   (make-pathname
```

Comment:
Can this be changed to (make-pathname :name name :defaults path) instead?

Reply:
Addressed in https://github.com/xach/quickproject/pull/16